src/VSCodeExtension/README.md

```diff
@@ -1,12 +1,10 @@
-# Microsoft Quantum Development Kit Preview
+# Microsoft Quantum Development Kit Preview for .NET6
 
-Thank you for your interest in Microsoft's Quantum Development Kit for Visual Studio Code preview.
+This is a **preview version** of the Microsoft Quantum Development Kit extension for Visual Studio based on .NET6.0 components.
 
-The Quantum Development Kit contains the tools you'll need to build your own quantum computing programs and experiments.
+We're actively working on supporting .NET6.0 in our VS Code extension. This support is still under development and this extension should be considered experimental.
```

Comment:
```suggestion
The Microsoft Quantum Development Kit extension supporting .NET 6.0 is under active development and should be considered experimental.
```

Reply:
alternatively, "This extension is under active...."
Essentially the suggestion here is to remove possessive language.